src/demo/demo.ejs

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
I'm guessing you are setting this because of an AXE linter warning in your IDE. 
The SDK's language is actually determined at initialisation stage in the SDK config so hard coding it as English in the HTML element isn't always strictly true as the demo app itself could also be configured to be in a different language. In production this would be down to the integrator's host app to manage.

Reply:
Okay, will revert it then and configure my IDE to not warn me anymore.